tools/toolchain/clang_lto_pgo.sh

```diff
@@ -0,0 +1,86 @@
+#!/bin/bash -uex
+
+# Which Scylla branch to train on
+SCYLLA_BRANCH=master
+
+# Which LLVM release to build
+LLVM_TAG=llvmorg-15.0.7
+
+if [ -d "/optimized_clang" ];
+then
+	DIR="/optimized_clang"
+else
+	DIR="${PWD}/optimized_clang"
+	mkdir -p "${DIR}"
+fi
+cd "${DIR}"
+
+# Download the scylla codebase for training.
+# Scylla's dependencies are already installed.
+git clone https://github.com/scylladb/scylla --branch ${SCYLLA_BRANCH} --depth=1
+git -C scylla submodule update --init --depth=1
```

Comment:
Please use the workspace tree, so this doesn't break when the compiler and source tree diverge.